src/main/java/teammates/common/datatransfer/AccountAttributes.java

```diff
@@ -65,6 +65,22 @@ public AccountAttributes(String googleId, String name, boolean isInstructor,
         this.studentProfile.googleId = this.googleId;
     }
     
+    /**
+     * Gets a deep copy of this object.
+     */
+    public AccountAttributes getCopy() {
+        boolean isStudentProfileNull = this.studentProfile == null;
+        if (isStudentProfileNull) {
+            this.studentProfile = new StudentProfileAttributes();
+        }
+        AccountAttributes copy = new AccountAttributes(this.toEntity());
+        if (isStudentProfileNull) {
```

Comment:
Hmm ok so please add a comment so that someone won't remove this by accident in the near future